src/classes/Callable_API.cls

```diff
@@ -82,6 +82,10 @@ global with sharing class Callable_API implements System.Callable {
 
                 return JSON.serialize(rollupDefs,false);
 
+            } when 'crlp.iscrlpenabled' {
```

Comment:
@force2b , @seanpat09 : any objection to the Callable method name?

Reply:
Good for me.